packages/accessibility/global.d.ts

```diff
@@ -0,0 +1,8 @@
+declare namespace PixiGlobalMixins
```

Comment:
Why not just call this namespace `PIXI`?

Reply:
I agree to change the name to whatever you want except `PIXI`: this thing does not actually exists, its only a TS hack. I dont want it to collide with actual global namespace if we'll have any. 

Also people will type `PIXI.` in IDE, see stub interfaces and they wont understand whats going on.

Reply:
Mixins?

Reply:
GlobalMixins sounds better IMO

Reply:
GlobalMixins it is.